components/datatypes/browserdetails.schema.json

```diff
@@ -117,6 +117,64 @@
           "meta:descriptionId": "browserdetails##xdm:userAgent##description##80421"
         }
       }
+    },
+    "user-agent-client-hints": {
+      "properties": {
+        "xdm:brands": {
```

Comment:
As discussed, please add an object Property here "userAgentClientHints" and nest all the other properties under this.